test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -5578,6 +5578,156 @@ public void PackCommand_PackageFromNuspecWithFrameworkReferences_WithDuplicateEn
             }
         }
 
+        [Fact]
+        public void PackCommand_PackIcon_HappyPath_Succeed()
+        {
+            var iconTestSourceDir = new IconTestSourceDirectory("icon.jpg", "icon.jpg", 6);
+            TestPackIcon(iconTestSourceDir);
+        }
+
+        [Fact]
+        public void PackCommand_PackIcon_ImplicitFile_Succeed()
```

Comment:
Either I don't understand what this test is supposed to be testing, or I disagree that it should work in this way. If the file contains `content\icon.jpg` and the nuspec specifies the icon as `icon.jpg`, this should fail because the paths do not match.

Reply:
The nuspec that createss this test is:

```
<package>
  <metadata>
    <id>iconPackage</id>
    <version>5.2.0</version>
    <authors>Author1, author2</authors>
    <description>Sample icon description</description>
    <icon>icon.jpg</icon>
  </metadata>
  <files>
    <file src="content\*" />
  </files>
</package>
```
The `<file/>` entry with the `*`  puts all the content files in the package root. License files works that way

Reply:
ah yes, I keep forgetting that it works that way. I find it completely unintuitive :(